lib/internal/test_runner/test.js

```diff
@@ -359,6 +370,21 @@ class Test extends AsyncResource {
         column: loc[1],
         file: loc[2],
       };
+
+      if (sourceMaps === true) {
```

Comment:
Why checking `=== true` and not using directly `sourceMaps`?

`if (sourceMaps) {`